README.md

```diff
@@ -102,7 +102,7 @@ With docker compose:
 version: '3'
 services:
   gpodder2go:
-    image: oxtyped/gpodder2go:latest
+    image: ghcr.io/oxtyped/gpodder2go:latest
```

Comment:
thanks for this!

while you're at this, I think we should make it `main` instead of `latest`. We're not tagging `latest` based on the workflow.yml